content/news/04-clash122.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Clash 1.2.2 and future plans"
+date: "2020-06-14"
+description: "Bugs bugs, and future plans"
+disable_comments: false
+author: "martijnbastiaan"
+authorbox: true # Optional, enable authorbox for specific post
+summary: Last Friday we released Clash 1.2.2 on GitHub. In big part thanks to our lovely community, we've found and fixed many bugs. Thanks to some patches, Clash even runs anywhere from 2-20% faster for some common design patterns! With this blogpost I'd like to highlight some of the progress we've made since the 1.2 release, and elaborate on our future plans.
+toc: false
+mathjax: false
+---
+
+Hi everyone!
+
+Last Friday we released Clash 1.2.2 on GitHub. In big part thanks to our lovely community, we've found and fixed many bugs. Thanks to some patches, Clash even runs anywhere from 2-20% faster for some common design patterns! Of course, the full list of changes can be found in [the CHANGELOG](https://github.com/clash-lang/clash-compiler/releases/tag/v1.2.2) over at GitHub. With this blogpost I'd like to highlight some of the progress we've made since the 1.2 release, and elaborate on our future plans. 
```

Comment:
```suggestion
Last Friday we cut version 1.2.2 of Clash on [GitHub](https://github.com/clash-lang/clash-compiler/releases/tag/v1.2.2), uploaded a source release on [Hackage](http://hackage.haskell.org/package/clash-ghc-1.2.2) and a binary release on [Snapcraft](https://snapcraft.io/clash). In big part thanks to our lovely community, we've found and fixed many bugs. Thanks to some patches, Clash even runs anywhere from 2-20% faster for some common design patterns! Of course, the full list of changes can be found in [the CHANGELOG](https://github.com/clash-lang/clash-compiler/releases/tag/v1.2.2) over at GitHub. With this blogpost we'd like to highlight some of the progress we've made since the 1.2 release, and elaborate on our future plans. 
```

Reply:
"we cut version 1.2.2 on GitHub" -> "we _put_"

I can't propose a change to a proposed change by the looks of it :-)